src/plugins/console/server/config.ts

```diff
@@ -31,7 +31,7 @@ const schemaLatest = schema.object(
         defaultValue: 'stack',
       }),
     }),
-    dev: schema.object({ enableMonaco: schema.boolean({ defaultValue: false }) }),
+    dev: schema.object({ enableMonaco: schema.boolean({ defaultValue: true }) }),
```

Comment:
Huge, this has been a loong time coming! Great work @yuliacech !

Reply:
Thank you, @jloleysens! We are super excited to get the new editor in front of real users!